### ISSUE TITLE
[Enhancement] Allow nesting of data elements in kirby-label in kirby-item

### ISSUE THREAD

**Issue by troelslenda** [kirbydesign/designsystem#3847]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 
I can't check ☝ this :)

In the cookbook, this example of the use of kirby-item

![Image](https://github.com/user-attachments/assets/17cf4e08-862d-4d0c-bffb-3567fe4c65cc)

has inspired me to create this list

![Image](https://github.com/user-attachments/assets/3e1904d7-caf8-4398-b2dc-ce8537e6153e)

It's awesome that this is actually possible with a kirby item. For semantic reasons the value written in percent is wrapped in a `data` element as suggested by kirby, this however has the side effect of introducing text truncate with ellipsis.

I believe that the reason for this is because https://github.com/kirbydesign/designsystem/blob/develop/libs/core/src/scss/base/_item-typography.scss#L3 includes the `data` element.

I can solve my issue in two ways. Overriding the CSS properties `text-overflow` and `overflow` or i can simply choose to not use a `data` element, but maybe just a div, but I cant figure out the reason why `data` is in this list in `_item-typography.scss` in the firstplace. Maybe it could be removed?

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
